app.py

```diff
@@ -27,21 +27,51 @@
 # class Health(db.Model):
 #     id = db.Column(db.Integer, primary_key=True)
 #     username = db.Column(db.String(80))
-db.create_all()
+# db.create_all()
 
 bp = flask.Blueprint("bp", __name__, template_folder="./build")
-app.register_blueprint(bp)
+
+#main landing page when app is launched
+@app.route("/", methods=["GET", "POST"])
```

Comment:
I think separating "GET" and "POST" into separate methods is a good practice, but for now keeping both in one method is okay.